apps/application-system/api/infra/application-system-api.ts

```diff
@@ -74,6 +74,17 @@ export const serviceSetup = (services: {
   service('application-system-api')
     .namespace(namespace)
     .serviceAccount(serviceAccount)
+    .features({
+      INAO_Client: {
```

Comment:
I cannot find where this flag is used in the code. Unconditional use of feature-specific environment variable can be dangerous. For more info on server-side features, please see - https://docs.devland.is/libs/nest/config#server-side-features

Reply:
The flag isn't used in the code. This is because these variables doesn't exist in the AWS parameter store and this is to prevent them to be configured there. I was copying Snorri's work in this PR: https://github.com/island-is/island.is/pull/7685

Reply:
That was the part where we configure the deployment and the feature. But The code needs to respect the flag and not throw because of missing env variables, which is what happened during the last release to prod.